docs/01-module-block-1/02-learning-goals.adoc

```diff
@@ -1,25 +1,44 @@
 === {learning-goals}
 
 // tag::DE[]
-Die Teilnehmer:Innen …
+In diesem Kapitel erreichen die Teilnehmer:innen die folgenden Lernziele:
 
 [[LZ-1-1]]
-==== LZ 1-1: Wissen wie man Künstliche Intelligenz definiert und die Anordnung zu Machine Learning, Data Science, Deep Learning, Generative AI.
+==== LZ 1-1: Definition Künstliche Intelligenz inklusive der Einordnung von Machine Learning, Data Science, Deep Learning, Generative AI
+
+Die Teilnehmer:innen wissen wie man Künstliche Intelligenz definiert und wie Machine Learning, Data Science, Deep Learning, Generative AI darin eingeordnet werden.
```

Comment:
Die Teilnehmer:innen wissen, wie ... 

Da müsste jeweils überall ein Komma mit dazu. Ich müsst auch nicht nochmal den Text des Lernziels als Wiederholung in einen Satz packen, wenn es keinen Mehrwert gibt. Dann ist das Lernziel halt vorerst leer, bis ihr mehr Info habt, was ihr da noch mit reinpacken wollt.